alphagov/govuk-coronavirus-find-support
# Add Support advice links

Dependency
----------------

This PR is branched off of branch `csv-to-yaml`
Review blocked until this PR had been merged:
[Rake task: Export results links as YAML](https://github.com/alphagov/govuk-coronavirus-find-support/pull/222)

What
----

- Adds content (automatically imported from spreadsheet) for support and advice links)

Why
----

Requested content update for the team.

How to review
-------------

**Devs*
- Check Rubocop Passes
- Check the quality checks pass

**Content Designers**
- Compare each "commit" to state in spreadsheet + google doc, check it reflects what you asked for
(let's just do this the first time to make sure everything worked!)

Links
-----

- Trello [Results page update (3/3): Rearrange content](https://trello.com/c/rbRyihX0/401-results-page-update-3-3-rearrange-content)
- Spreadsheet - [Find Support - Results Content Sheet](https://docs.google.com/spreadsheets/d/11jtiwMovC9736F7ZV9k3gDbOpMMGqnSsiRmlD0JFCpQ/edit#gid=0)
